src/lua/error.lua

```diff
@@ -39,6 +39,9 @@ error_set_prev(struct error *e, struct error *prev);
 const char *
 box_error_custom_type(const struct error *e);
 
+void
+error_ref(struct error *e);
```

Comment:
To be able to use `error_ref` in Lua via FFI you should export `error_ref` symbol. Add it to `exports.h` file, respecting the alphabetic order of symbols there.